module/VuFind/src/VuFind/Content/Covers/ObalkyKnih.php

```diff
@@ -65,6 +65,7 @@ public function __construct(ObalkyKnihService $service)
         $this->supportsNbn = true;
         $this->cacheAllowed = false;
         $this->directUrls = true;
+        $this->mandatoryBacklinks = true;
```

Comment:
Should we add a note in config.ini (along with the other requirements for ObalkyKnih around [this line](https://github.com/vufind-org/vufind/blob/dev/config/vufind/config.ini#L882)) to indicate that it now only works with ajaxCovers = true?

Reply:
I added  a note into config.ini